<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vaccipy - Kontaktdaten</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="header_label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>vaccipy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="subheader_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gebe Bitte deine Persöhnliche Daten ein</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="main_grid">
     <item row="1" column="1">
      <layout class="QFormLayout" name="kontakdaten_layout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="plz_impfzent_label">
         <property name="text">
          <string>PLZ's der Impfzentren:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="code_label">
         <property name="text">
          <string>Code:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="i_code_impfzentren">
         <property name="inputMask">
          <string>NNNN-NNNN-NNNN</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="placeholderText">
          <string>A1C3-D2FG-4IJK</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="anrede_label">
         <property name="text">
          <string>Anrede:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="i_anrede_combo_box">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string extracomment="Bitte Wählen">Bitte Wählen</string>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <item>
          <property name="text">
           <string>Bitte Wählen</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Frau</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Herr</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="vorname_label">
         <property name="text">
          <string>Vorname:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="i_vorname">
         <property name="placeholderText">
          <string>Max</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="nachname_label">
         <property name="text">
          <string>Nachname:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="i_nachname">
         <property name="placeholderText">
          <string>Mustermann</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="strasse_label">
         <property name="text">
          <string>Straße:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="strasse_grid_layout">
         <item row="0" column="2">
          <widget class="QLabel" name="hausnummer_label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Hausnummer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="i_strasse">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="i_hausnummer">
           <property name="inputMask">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_plz_wohnort">
         <property name="text">
          <string>PLZ Wohnort:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QGridLayout" name="plz_grid_layout">
         <item row="0" column="3">
          <widget class="QLineEdit" name="i_wohnort">
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="wohnort_label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Wohnort:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="i_plz_wohnort">
           <property name="inputMask">
            <string>99999</string>
           </property>
           <property name="placeholderText">
            <string>88045</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="telefon_label">
         <property name="text">
          <string>Telefonnummer:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="i_telefon">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>+49</string>
         </property>
         <property name="placeholderText">
          <string>+4917299334455</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="mail_label">
         <property name="text">
          <string>Mail:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="i_mail">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="plz_zentren_grid_layout">
         <item row="0" column="0">
          <widget class="QLineEdit" name="i_plz_impfzentren">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Beispiel: 68163, 69124, 69469</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="b_impfzentren_waehlen">
           <property name="text">
            <string>Impfzentren auswählen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>i_code_impfzentren</tabstop>
  <tabstop>i_anrede_combo_box</tabstop>
  <tabstop>i_vorname</tabstop>
  <tabstop>i_nachname</tabstop>
  <tabstop>i_strasse</tabstop>
  <tabstop>i_hausnummer</tabstop>
  <tabstop>i_plz_wohnort</tabstop>
  <tabstop>i_wohnort</tabstop>
  <tabstop>i_telefon</tabstop>
  <tabstop>i_mail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
